libnavigation-core/src/main/java/com/mapbox/navigation/core/lifecycle/MapboxNavigationOwner.kt

```diff
@@ -60,6 +60,8 @@ internal class MapboxNavigationOwner {
             attached = false
             services.forEach { it.onDetached(mapboxNavigation!!) }
             MapboxNavigationProvider.destroy()
+            mapboxNavigation = null
+            logI("disabled ${services.size} observers", LOG_CATEGORY)
```

Comment:
Adding this log because it is useful to figure out how many services are attached when you call disable.